keystore.go

```diff
@@ -2,13 +2,9 @@ package keystore
 
 import (
 	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"strings"
 
-	logging "github.com/ipfs/go-log"
-	ci "github.com/libp2p/go-libp2p-crypto"
+	logging "gx/ipfs/QmRb5jh8z2E8hMGN2tkvs1yHynUanqnZ3UeKwgN1i9P1F8/go-log"
+	ci "gx/ipfs/QmaPbCnUMBohSGo3KnxEa2bHqyJVVeEEcwtqJAYxerieBo/go-libp2p-crypto"
```

Comment:
gx paths in small deps are not needed, we use it mostly in go-ipfs to enforce the versions.

Just un-rewrite it and it think it is good to go.